Squablr/View Controllers/UserFeedViewController.m

```diff
@@ -113,15 +98,16 @@ - (void)swipeableView:(ZLSwipeableView *)swipeableView
 #pragma mark - ZLSwipeableViewDataSource
 
 - (UIView *)nextViewForSwipeableView:(ZLSwipeableView *)swipeableView {
-
-    CardView *view = [[CardView alloc] initWithFrame:swipeableView.bounds];
-    view.backgroundColor = [UIColor blueColor];
-    // Attempt to load user info into the card view
-    view.imageView.file = self.userProfilePhotos[0];
-    [view.imageView loadInBackground];
-    [self.userProfilePhotos removeObjectAtIndex:0];
-
-    return view;
+    // call the line below with the initwithProfile
+    if (self.profileIndex < self.userProfileInfo.count) {
```

Comment:
Can we name this something a little more specific? userProfileInfo is a little vague, like is that the current user's profile info or the profile feed the user is swiping on?